src/ui/MentionInputWidget.css

```diff
@@ -0,0 +1,64 @@
+.mentions {
+    /* unused */
```

Comment:
use SCSS
All wrapped in container class `mx-widget-mentions` to prevent conflicts with other widget and styling in the app